README.rst

```diff
@@ -7,7 +7,7 @@
 .. image:: https://img.shields.io/readthedocs/twine
    :target: https://twine.readthedocs.io
 
-.. image:: https://img.shields.io/github/workflow/status/pypa/twine/Main
+.. image:: https://img.shields.io/github/actions/workflow/status/pypa/twine/codeql-analysis.yml?branch=main
```

Comment:
I'm pretty sure the intent is to show the "Main" CI workflow, from the `main` branch.

```suggestion
.. image:: https://img.shields.io/github/actions/workflow/status/pypa/twine/main.yml?branch=main
```